indy_node/server/domain_req_handler.py

```diff
@@ -142,6 +138,10 @@ def _validateNym(self, req: Request):
         origin = req.identifier
         op = req.operation
 
+        s, reason = self.canNymRequestBeProcessed(origin, op)
```

Comment:
Do we need this call at all? Doesn't `_validateExistingNym` already do similar validation based on `Authoriser`?

Reply:
Yeah, seems like it is double-check, have removed.